modules/zookeeper/single_docker_beanstalk.tf

```diff
@@ -0,0 +1,237 @@
+data "aws_elastic_beanstalk_solution_stack" "single_docker" {
+  most_recent = true
+  name_regex  = "^64bit Amazon Linux (.*) running Docker (.*)$"
+}
+
+data "aws_region" "current" {
+  current = true
+}
+
+resource "aws_elastic_beanstalk_environment" "zookeeper" {
+  name                = "${var.environment_name}"
+  cname_prefix        = "${var.environment_cname}"
+  application         = "${var.application_name}"
+  version_label       = "${var.version_label}"
+  solution_stack_name = "${data.aws_elastic_beanstalk_solution_stack.single_docker.name}"
+
+  setting {
+    namespace = "aws:autoscaling:asg"
+    name      = "MinSize"
+    value     = 3
+  }
+
+  setting {
+    namespace = "aws:autoscaling:asg"
+    name      = "MaxSize"
+    value     = 3
+  }
+
```

Comment:
Same question from the solr module about moving these from variables, just to put a pin in it here.

Reply:
See response on the solr PR. This one is actually even more of a touchy application when nodes coming and going fwiw